.release-notes/next-unwrap.md

```diff
@@ -0,0 +1,105 @@
+## Make working with Promise[Promise[B]] chains easier
+
+We've added a new method `flatten_next` to `Promise` in order to make working with promises of promises easier.
+
+`flatten_next` is a companion to `next`. It operates in an identical fashion except for the type of the fulfilled promise. Whereas `next` returns a type `B`, `flatten_next` returns `Promise[B]`.
```

Comment:
Strictly speaking, they both return `Promise[B]` - what differs is not the return type of these methods, but rather the return type of the fulfill function they take as an argument. 

```suggestion
`flatten_next` is a companion to `next`. It operates in an identical fashion except for the type of the fulfilled promise. Whereas `next` takes a function that returns a type `B`, `flatten_next` takes a function that returns `Promise[B]`.
```

Reply:
I made the same edit in the docstring that had the same language.